.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: node_js
+
+sudo: false
+
+node_js:
+  - "0.10"
+  - "0.12"
+  - "iojs"
```

Comment:
Yeah, Node 0.10 (maintenance), Node 0.12 (stable), and iojs (stable). We can explicitly test on iojs v1 and v2 as well, but who cares about iojs v1. Plus the builds mostly run in parallel, so there isn't a big time hit for specifying multiple versions.

And if I understand Travis files correctly, the `matrix` below means that Node 0.10/0.12 builds have to pass, but iojs can fail (without breaking the whole build).

http://docs.travis-ci.com/user/languages/javascript-with-nodejs/